ai-quick-actions/cli.md

```diff
@@ -0,0 +1,821 @@
+```
+
+<details>
+  <summary>CLI Output</summary>
+
+```console
+{
+    "id": "ocid1.datasciencemodeldeployment.oc1.iad.<ocid>",
+    "display_name": "test_evaluation",
+    "aqua_service_model": true,
+    "state": "CREATING",
+    "description": null,
+    "created_on": "2024-02-03 21:21:31.952000+00:00",
+    "created_by": "ocid1.user.oc1..<ocid>",
+    "endpoint": "https://modeldeployment.us-ashburn-1.oci.customer-oci.com/ocid1.datasciencemodeldeployment.oc1.iad.<ocid>",
+    "console_link": "https://cloud.oracle.com/data-science/model-deployments/ocid1.datasciencemodeldeployment.oc1.iad.<ocid>?region=us-ashburn-1",
+    "shape_info": {
+        "instance_shape": "VM.Standard.E4.Flex",
+        "instance_count": 1,
+        "ocpus": 1.0,
+        "memory_in_gbs": 16.0
+    },
+    "tags": {
+        "aqua_service_model": "ocid1.datasciencemodel.oc1.iad.<ocid>#Llama-2-13b",
+        "OCI_AQUA": ""
+    }
+}  
+```
+</details>
+
+
+
+## List Model Evaluations
+
+### Description
+
+Lists all Aqua model evaluations within a specified compartment and/or project.
+
+### Usage
+
+```bash
+ads aqua evaluation list [OPTIONS]
+```
+
+### Required Parameters
+
+`--compartment_id [text]`
+
+The ID of the compartment in which the aqua model evaluations are available.
+
+
+### Example
+
+```bash
+ads aqua evaluation list --compartment_id ocid1.compartment.oc1..<ocid>
+```
+
+<details>
+  <summary>CLI Output</summary>
+
+```console
+{
+    "id": "ocid1.datasciencemodel.oc1.iad.<ocid>",
+    "name": "test-eval",
+    "console_url": "https://cloud.oracle.com/data-science/models/ocid1.datasciencemodel.oc1.iad.<ocid>?region=us-ashburn-1",
+    "lifecycle_state": "FAILED",
+    "lifecycle_details": "",
+    "time_created": "2024-03-18T00:31:28.026000+00:00",
+    "tags": {
+        "aqua_evaluation": "aqua_evaluation"
+    },
+    "experiment": {
+        "id": "ocid1.datasciencemodelversionset.oc1.iad.<ocid>",
+        "name": "experiment_name",
+        "url": "https://cloud.oracle.com/data-science/model-version-sets/ocid1.datasciencemodelversionset.oc1.iad.<ocid>?region=us-ashburn-1"
+    },
+    "source": {
+        "id": "ocid1.datasciencemodeldeploymentint.oc1.iad.ocid",
+        "name": "mistral-classifier",
+        "url": "https://cloud.oracle.com/data-science/model-deployments/ocid1.datasciencemodeldeploymentint.oc1.iad.<ocid>?region=us-ashburn-1"
+    },
+    "job": {
+        "id": "ocid1.datasciencejobrunint.oc1.iad.<ocid>",
+        "name": "test-eval",
+        "url": "https://cloud.oracle.com/data-science/job-runs/ocid1.datasciencejobrunint.oc1.iad.<ocid>?region=us-ashburn-1"
+    },
+    "parameters": {
+        "max_tokens": 500,
+        "top_p": 1,
+        "top_k": 50,
+        "temperature": 0.7,
+        "presence_penalty": 0,
+        "frequency_penalty": 0,
+        "stop": [],
+        "shape": "VM.Standard.E3.Flex",
+        "dataset_path": "oci://aqua-eval-test-bucket@namespace/dataset/evaluation-sample-with-sys-message.jsonl",
```

Comment:
I guess the real path should be removed?

Reply:
yea this is a made-up path as well, but I updated it to `oci://aqua-eval-test-bucket@namespace/dataset/evaluation-sample.jsonl`